channels/mx.m3u

```diff
@@ -43,8 +37,6 @@ http://live.canaloncelive.tv:1935/livepkgr2/smil:internacional.smil/playlist.m3u
 https://180ce25c70.site.internapcdn.net/ryta_live/chunklist_b481072.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/NAgRCyY.png" group-title="",Canal Once
 https://live2.canaloncelive.tv/livepkgr2/smil:internacional.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/857zxbL.png" group-title="",Capacita TV
-https://srvstream01.televisioneducativa.gob.mx/hls/capacitatv.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC output:
<img width="741" alt="Screenshot 2019-10-12 at 20 16 08" src="https://user-images.githubusercontent.com/7253922/66705275-29ac9c00-ed2d-11e9-9dd6-e789046c1850.png">

Reply:
Still works for me